data/pages/data-privacy-de.md

```diff
@@ -8,7 +8,7 @@ language: de
 1. Im Folgenden informieren wir über die Erhebung personenbezogener Daten bei Nutzung unserer Website. Personenbezogene
    Daten sind alle Daten, die auf Sie persönlich beziehbar sind, z. B. Name, Adresse, E-Mail-Adressen, Nutzerverhalten.
 2. Verantwortlicher gem. Art. 4 Abs. 7 EU-Datenschutz-Grundverordnung (DSGVO) ist Satellytes Digital Consulting GmbH,
-   Carola-Neher-Straße 10, 81379 München, info@satellytes.com (siehe [Impressum](/de/imprint/)).
+   Carola-Neher-Straße 10, 81379 München, info@satellytes.com (siehe [Impressum](/imprint/)).
```

Comment:
Yes, this link is now rendered as /de/imprint:
![image](https://user-images.githubusercontent.com/78978542/161777474-f4226579-c0f8-44e7-b661-fb568b68d602.png)

Reply:
Yeah, but this is the german data privacy page, which should link to `/de/imprint` and not `/imprint` like in your change. Do I miss something? 😄

Reply:
Yes, but the thing in #293 is, that all links on all our pages should automatically append "de" if language === "de".

But somehow, this did not work in markdown, but by using the Link component in our custom Components this Automation also works here.

I changed the "german" links in markdown, so that they do not get rendered as "/de/de/imprint" on german.

Reply:
You are right, thanks for the clarification 👏